tools/build_pytorch_libs.bat

```diff
@@ -135,7 +135,7 @@ goto:eof
                   -DCMAKE_INSTALL_PREFIX="%INSTALL_DIR%" ^
                   -DCMAKE_C_FLAGS="%C_FLAGS%" ^
                   -DCMAKE_SHARED_LINKER_FLAGS="%LINK_FLAGS%" ^
-                  -DCMAKE_CXX_FLAGS="%C_FLAGS% %CPP_FLAGS%" ^
+                  -DCMAKE_CXX_FLAGS="/MT %C_FLAGS% %CPP_FLAGS%" ^
```

Comment:
Why is /MT needed here? According to MSDN, /MT causes the application to use the multithread, static version of the run-time library. It has nothing to do with build speed.
--


`